src/components/sidebar/sidebar.style.js

```diff
@@ -42,8 +44,20 @@ const SidebarStyle = styled.div`
   }
 `;
 
+export const SidebarContentBox = styled(Box)``;
+
+SidebarContentBox.defaultProps = {
+  mr: -4,
```

Comment:
There must be a better way to do this than using negative margins - we really should avoid using negative margins